build.go

```diff
@@ -47,7 +47,13 @@ func Build(entryResolver EntryResolver, dependencyManager DependencyManager, sbo
 		}
 
 		entry, allEntries := entryResolver.Resolve("node", context.Plan.Entries, priorities)
-		logger.Candidates(allEntries)
+		if entry.Name == "" && len(allEntries) == 0 {
+			logger.Process("Node no longer requested by plan")
+			return packit.BuildResult{}, nil
+		} else {
+			logger.Process("Resolving Node Engine version")
+			logger.Candidates(allEntries)
+		}
```

Comment:
Stylistically, most Go code doesn't explicitly leave the `else` condition when the `if` path returns.

```suggestion
		}
		logger.Process("Resolving Node Engine version")
		logger.Candidates(allEntries)
```

Reply:
Thanks for the review/suggestion.